src/GovUk.Education.ExploreEducationStatistics.Data.ViewModels/LocationViewModelBuilder.cs

```diff
@@ -10,7 +11,7 @@ public static class LocationViewModelBuilder
     public static Dictionary<GeographicLevel, List<LocationAttributeViewModel>> BuildLocationAttributeViewModels(
         IList<Location> locations,
         Dictionary<GeographicLevel, List<string>>? hierarchies,
-        Dictionary<GeographicLevel, Dictionary<string, GeoJson>>? geoJson = null)
+        Dictionary<GeographicLevel, Dictionary<string, BoundaryData>>? geoJson = null)
```

Comment:
Because we are renaming the type from `BoundaryData` to `GeoJson` I think you should consider renaming all the `geoJson` variables that occur in many places, not just here.